service/controller/resource/deployment/template/template.go

```diff
@@ -0,0 +1,30 @@
+package template
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/giantswarm/microerror"
+	"github.com/markbates/pkger"
+)
+
+// GetARMTemplate returns the ARM template reading a json file locally using pkger.
+func GetARMTemplate() (map[string]interface{}, error) {
```

Comment:
This is the function I created for reading files locally using `pkger`. It's copied and pasted in different resources. I didn't spent much time on code design since node pools is around the corner.